sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVParameters.scala

```diff
@@ -44,9 +44,9 @@ private[sql] case class CSVParameters(parameters: Map[String, String]) extends L
     }
   }
 
-  val delimiter = CSVTypeCast.toChar(parameters.getOrElse("delimiter", ","))
+  val seq = CSVTypeCast.toChar(parameters.getOrElse("seq", ","))
```

Comment:
I think it is "sep", not "seq".

Also I like the old name for the variable better. We should just rename the name for the option. And for backward compatibility, we should accept "delimiter" if "sep" is not set.